src/pool/srv_iv.c

```diff
@@ -296,13 +296,20 @@ pool_iv_conn_ent_size(size_t cred_size)
 	       pool_iv_conn_size(cred_size);
 }
 
+static bool
+pool_iv_conn_valid(struct pool_iv_conn *conn, char *end)
+{
+	return (conn->pic_creds <= end) &&
+	       ((char *)conn + pool_iv_conn_size(conn->pic_cred_size) <= end);
```

Comment:
only conn + pool_iv_conn_size() is enough here?

Reply:
I was think that when "conn->pic_creds <= end" not true, need not calculate pool_iv_conn_size(conn->pic_cred_size) as that is possibly incorrect already.